src/pages/components/button/style.mdx

```diff
@@ -6,207 +6,435 @@ description:
+
+| Element   | Property         | Color token         |
+| --------- | ---------------- | ------------------- |
+| Label     | text color       | `$text-on-color`    |
+| Icon      | svg              | `$icon-on-color`    |
+| Container | background-color | `$button-secondary` |
 
 <Row>
 <Column colLg={8}>
 
-![Tertiary button states and color](images/button-style-tertiary.png)
+<Tabs>
+
+<Tab label="Button with icon">
+
+![Secondary button color](images/button-style-secondary.png)
+
+</Tab>
+
+<Tab label="Icon only button">
+
+![Secondary icon only button color](images/button-style-secondary-icon-only.png)
+
+</Tab>
+
+</Tabs>
 
 </Column>
 </Row>
 
-### Ghost button
-
-| Element          | Property         | Color token           |
-| ---------------- | ---------------- | --------------------- |
-| Label            | text color       | `$link-primary`       |
-| Icon             | svg              | `$link-primary`       |
-| Container        | background-color | Transparent           |
-| Label:hover      | text color       | `$link-primary-hover` |
-| Icon:hover       | svg              | `$link-primary-hover` |
-| Container:hover  | background-color | `$background-hover`   |
-| Container:active | background-color | `$background-active`  |
-| Container:focus  | border           | `$focus`              |
-| Label:disabled   | text color       | `$text-disabled`      |
-| Label:disabled   | svg              | `$icon-disabled`      |
+<Title>Secondary button interactive state color</Title>
+
+| State    | Element   | Property         | Color token                |
+| -------- | --------- | ---------------- | -------------------------- |
+| Hover    | Container | background-color | `$button-secondary-hover`  |
+| Focus    | Container | border           | `$focus`                   |
+|          |           | inset            | `$focus-inset`             |
+| Active   | Container | background-color | `$button-secondary-active` |
+| Disabled | Label     | text-color       | `$text-on-color-disabled`  |
+|          | Icon      | svg              | `$icon-on-color-disabled`  |
+|          | Container | background-color | `$button-disabled`         |
 
 <Row>
 <Column colLg={8}>
 
-![Ghost button states and color](images/button-style-ghost.png)
+<Tabs>
+
+<Tab label="Button with icon">
+
+![Secondary button interactive states color](images/button-style-secondary-interactive-states.png)
+
+</Tab>
+
+<Tab label="Icon only button">
+
+![Secondary icon only button interactive states color](images/button-style-secondary-icon-only-interactive-states.png)
+
+</Tab>
+
+</Tabs>
 
 </Column>
 </Row>
 
-<br />
+### Tertiary button color
 
-#### Ghost button – Icon only
+| Element   | Property         | Color token        |
+| --------- | ---------------- | ------------------ |
+| Label     | text color       | `$button-tertiary` |
+| Icon      | svg              | `$button-tertiary` |
+| Container | background-color | `transparent`      |
```

Comment:
Transparent does not need to be in an inline code snippet. There are other instances of this throughout the page.